pkgs/tools/graphics/metapixel/default.nix

```
@@ -0,0 +1,33 @@
+{stdenv, fetchFromGitHub, libpng, libjpeg, giflib, perl, pkg-config}:
+
+stdenv.mkDerivation {
+  pname = "metapixel";
+  version = "1.0.2";
+
+  src = fetchFromGitHub {
+    owner = "schani";
+    repo = "metapixel";
+    rev = "98ee9daa093b6c334941242e63f90b1c2876eb4f";
+    fetchSubmodules = true;
+    sha256 = "0r7n3a6bvcxkbpda4mwmrpicii09iql5z69nkjqygkwxw7ny3309";
+  };
+
+  makeFlags = [ "PREFIX=build" "metapixel" ];
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ libpng libjpeg giflib perl ];
```

```suggestion
  nativeBuildInputs = [ perl pkg-config ];
  buildInputs = [ libpng libjpeg giflib ];
```

I am somewhat sure this is correct.

---

If anything it goes into propagatedBuildInputs it's specifically required for the perl scripts which will be in /bin NOT while building. It's here so the shebang hook will replace it